conda/exceptions.py

```diff
@@ -757,6 +760,11 @@ def handle_exception(self, exc_val, exc_tb):
             return 1
         elif isinstance(exc_val, SystemExit):
             return exc_val.code
+        elif (isinstance(exc_val, (IOError, OSError))
```

Comment:
You can spell this `EnvironmentError`.

Reply:
Thanks! I didn't realize `EnvironmentError` was the python 2/3 agnostic way to do this.